reana_workflow_engine_serial/tasks.py

```diff
@@ -52,9 +52,6 @@ def initialize(workflow_workspace, operational_options):
     else:
         cache_enabled = False
 
-    # build workspace path
-    workflow_workspace = "{0}/{1}".format(SHARED_VOLUME_PATH, workflow_workspace)
```

Comment:
Minor: we can also remove the import of ` SHARED_VOLUME_PATH` since it's not used anymore here

Reply:
Should I change something in `reana-workflow-engine-snakemake`?

Reply:
Good catch, we will also have to remove similar line in [reana-workflow-engine-snakemake](https://github.com/reanahub/reana-workflow-engine-snakemake/blob/59fe4a279a8ebb6819e8424d6b2a7ee8b20c9684/reana_workflow_engine_snakemake/cli.py#L39)

Reply:
Thanks, I see [the PR](https://github.com/reanahub/reana-workflow-engine-snakemake/pull/4). It looks good to me but I don't have a deep knowledge of the workspace abstraction sprint. Feel free to review/merge it when needed.